code/modules/projectiles/guns/syringe_gun.dm

```diff
@@ -68,6 +68,9 @@
 
 /obj/item/gun/syringe/attackby(obj/item/A, mob/user, params, show_msg = TRUE)
 	if(istype(A, /obj/item/reagent_containers/syringe))
+		if(istype(A, /obj/item/reagent_containers/syringe/lethal))
+			to_chat(user, "<span class='warning'>[A] is too big to fit into [src].</span>")
+			return
```

Comment:
Could do a check of the max size of the syringe instead, for admemery?

Reply:
If it is admemery, I hope they are being responsible enough to trust them with the ability